base_layer/wallet/src/transaction_service/handle.rs

```diff
@@ -53,9 +53,10 @@ pub enum TransactionServiceRequest {
     GetAnyTransaction(TxId),
     SetBaseNodePublicKey(CommsPublicKey),
     SendTransaction((CommsPublicKey, MicroTari, MicroTari, String)),
+    SendOneSidedTransaction((CommsPublicKey, MicroTari, MicroTari, String)),
```

Comment:
nit: I see this is the same as SendTransaction, but in future, it would be better to use field names instead of a tuple.

The extra wrapping `()` are also not needed

Reply:
- `SendOneSidedTransaction` is specifically made into a separate interface from `SendTransaction`, otherwise, a differentiating parameter must be added to the interface to distinguish the two variants, with widespread impact. I believe this to be a cleaner solution.
- Tuple removed from the new and old interfaces